libraries/AP_Camera/AP_Camera_Logging.cpp

```diff
@@ -64,12 +65,22 @@ void AP_Camera_Backend::Write_CameraInfo(enum LogMessages msg, uint64_t timestam
 void AP_Camera_Backend::Write_Camera(uint64_t timestamp_us)
 {
     Write_CameraInfo(LOG_CAMERA_MSG, timestamp_us);
+    auto *mount = AP_Mount::get_singleton();
+    if (mount!= nullptr) {
+        // Assuming camera1 corresponds mount1 and camera2 corresponds mount2 an so on for others
+        mount->Write_Mount(_instance, LOG_MOUNT_MSG, timestamp_us);
+    }
```

Comment:
```suggestion
    }
#endif
```

Also, we still want to log this message even if the Camera library isn't compiled in.  We do want them correlated in time with the camera message,so I think this code does belong here, but we should also call the mount logging message within the mount library if the camera library isn't present.

Similarly elsewhere.

Reply:
Yes, I think we should log this mount message at 10hz but then also log an extra MNT message whenever the camera takes a picture.